webapp/src/main/java/com/box/l10n/mojito/security/WebSecurityConfig.java

```diff
@@ -33,7 +36,7 @@
 // TOOD(spring2) we don't use method level security, do we? remove?
 @EnableGlobalMethodSecurity(securedEnabled = true, mode = AdviceMode.ASPECTJ)
 @Configuration
-public class WebSecurityConfig extends WebSecurityConfigurerAdapter {
+public class WebSecurityConfig {
```

Comment:
General comment on the paths, Spring 3.x disables the `useTrailingSlashMatch` by default. I suspect we've probably be relying on this in tests so it might make sense to re-enable it globally. I'm unsure if anything needs to be done inside this class but keep it in mind if we run into issues later https://github.com/spring-projects/spring-boot/wiki/Spring-Boot-3.0-Migration-Guide#spring-mvc-and-webflux-url-matching-changes

Reply:
well observed, thanks for pointing out. 🙏